pkg/auth/agent_local_authenticator.go

```diff
@@ -78,6 +78,19 @@ func (a *AgentLocalAuthenticator) AuthAgentAuth(token string) (interface{}, erro
 		return nil, common.NewInfraError(http.StatusUnauthorized, err)
 	}
 
+	exp, found := claims["exp"].(float64)
+	if !found {
+		// exp claim is not found in the case of install workflow
+		return ocm.AdminPayload(), nil
```

Comment:
Even if `exp` is not found you should still test for the infraenv, right? I'd expect this checking for the expiration to happen after the infraenv check.

Reply:
ah , right